release-notes/1.3.0.md

```diff
@@ -0,0 +1,4 @@
+# 1.3.0
```

Comment:
Should this be 1.2.1? Feels like a patch update rather than introducing anything new...